monitoring/zabbix_link_template.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/python
+
+# (c) 2016, Michal Mankowski <michal.mankowski () intel.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from ansible.module_utils.basic import *
+try:
+    from pyzabbix import ZabbixAPI
+    HAS_ZABBIX = True
+except ImportError:
+    HAS_ZABBIX = False
+
+DOCUMENTATION = '''
+---
+module: zabbix_update
+short_description: Link/unlink given hosts list with given templates list
+description:
+  - Link/unlink given hosts list with given templates list
+
+version_added: "2.3"
+author: "Michal Mankowski (@MichalMankowski)"
+requirements:
+  - "python >= 2.6"
+  - "pyzabbix >= 0.7.2"
+options:
+  server_url:
+     description:
+         - Address of zabbix server
+     required: True
+  username:
+     description:
+         - name of user to login to zabbix
+     required: True
+  password:
+     description:
+         - password for user to login to zabbix
+     reuqierd: True
+  host:
+     description:
+         - list of hosts to update
+     required: True
+  template:
+     description:
+         - list of templates to link/unlink
+  state:
+     description:
+         - state present link templates, absent unlink templates
+     required: False
+'''
+
+EXAMPLES = '''
+---
+- name: link templates
+  local_action:
+    module: zabbix_update
+    server_url: "http://127.0.0.1"
+    username: "test"
+    password: "test"
+    template:
+    - "Template App HTTP Service"
+    host:
+    - "Zabbix server"
+    state: present
```

Comment:
please add some more examples e.g. for state=absent

and btw if you only add one item for a list, you can just pass it not being an item, e.g.

~~~yaml
- name: link templates
  local_action:
    module: zabbix_update
    server_url: "http://127.0.0.1"
    username: "test"
    password: "test"
    template: "Template App HTTP Service"
    host: "Zabbix server"
    state: present
~~~